### PR TITLE
Add Package: Neovim QT

### PR DESCRIPTION
**Pull request by techie-guy**
This adds Neovim QT, A QT GUI for Neovim
https://github.com/equalsraf/neovim-qt

Requested by me;) #16672 

The Package is not ready!

### CODE REVIEW

File: mingw-w64-neovim-qt/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,60 @@
+_realname=neovim-qt
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=0.2.17
+pkgrel=1
+pkgdesc="Neovim client library and GUI, in Qt5. (mingw-w64)"
+arch=('any')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32')
+url='https://github.com/equalsraf/neovim-qt'
+license=('ISC')
```
Comment (kmilos) [inline mingw-w64-neovim-qt/PKGBUILD:10]: 'spdx:ISC'

Comment (techie-guy) [reply mingw-w64-neovim-qt/PKGBUILD:10]: 👍

Comment (techie-guy) [reply mingw-w64-neovim-qt/PKGBUILD:10]: I'll add that.

**Comment by kmilos** [comment]
Also, please send any patches upstream whenever possible, then you can reference and download it by link instead of maintaining separately here.

File: mingw-w64-neovim-qt/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,60 @@
+_realname=neovim-qt
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=0.2.17
+pkgrel=1
+pkgdesc="Neovim client library and GUI, in Qt5. (mingw-w64)"
+arch=('any')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32')
```
Comment (kmilos) [inline mingw-w64-neovim-qt/PKGBUILD:8]: Could also add clangarm64 here?

File: mingw-w64-neovim-qt/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+url='https://github.com/equalsraf/neovim-qt'
+license=('ISC')
+makedepends=("${MINGW_PACKAGE_PREFIX}-cmake"
+             "${MINGW_PACKAGE_PREFIX}-ninja"
+             "${MINGW_PACKAGE_PREFIX}-cc"
+	     "${MINGW_PACKAGE_PREFIX}-qt5-base"
```
Comment (kmilos) [inline mingw-w64-neovim-qt/PKGBUILD:14]: Looks like you want 'qt5-svg' instead? Is it only a 'makedepends' or a runtime 'depends' as well? It's a bit weird there no runtime deps for this package...

Comment (techie-guy) [reply mingw-w64-neovim-qt/PKGBUILD]: I replaced `qt5-base` with `qt5-svg`.
I don't know if there are any runtime deps. Though, you have to set `NVIM_QT_RUNTIME_PATH`
https://github.com/equalsraf/neovim-qt#building-from-source

Comment (kmilos) [reply mingw-w64-neovim-qt/PKGBUILD]: `The nvim binary must be in your $PATH.`

So neovim is not just a 'makedepends', it's actually a 'depends'. I expect the same for qt5-base (if not qt5-svg)...

There is no need to list the in both places, 'depends' also covers make requirements...

**Pull request merged**